pcsx2/Recording/InputRecording.cpp

```diff
@@ -63,28 +63,51 @@ void SaveStateBase::InputRecordingFreeze()
 
 InputRecording g_InputRecording;
 
-InputRecording::InputRecording()
+InputRecording::~InputRecording()
 {
-	// NOTE - No multi-tap support, only two controllers
-	padData[CONTROLLER_PORT_ONE] = new PadData();
-	padData[CONTROLLER_PORT_TWO] = new PadData();
+	// As to not attempt a double delete from WX
+	newRecordingFrame.release();
+	openFileDialog.release();
```

Comment:
What's the scenario that leads to this double delete you speak of?  Once the object is destroyed these smart pointers should get automatically cleaned up should they not?

Reply:
When the new operator is called on a wxWindow object, the point of the new object gets sent into a list of that all wxWindow pointers. WX deletes every pointer in that list, so if the smart pointer deletes it first...